suite/tests/client-interface/drreg-test.dll.c

```diff
@@ -1154,7 +1154,6 @@ event_exit(void)
         drreg_exit() != DRREG_SUCCESS)
         CHECK(false, "exit failed");
 
-    drx_exit();
```

Comment:
Please add the comment suggested on the PR's main thread, about why we need to skip drx_exit and drx_init calls.